docs/modules/ROOT/pages/how-tos/_nav.adoc

```diff
@@ -9,6 +9,7 @@
 *** xref:how-tos/configuring/component-nudges.adoc[Defining component relationships]
 *** xref:how-tos/configuring/rerunning.adoc[Retriggering build pipelines]
 *** xref:how-tos/configuring/redundant-rebuilds.adoc[Preventing redundant rebuilds]
+*** xref:how-tos/configuring/build-with-args.adoc[Passing build arguments]
```

Comment:
```suggestion
*** xref:how-tos/configuring/build-with-args.adoc[Passing buildah build-args]
```

I would call it something like this. "Passing build arguments" is very generic, when I read it the first thing that comes to mind is "oh, this is probably about Tekton params"